docs/_chapters/650-windows.md

```diff
@@ -0,0 +1,70 @@
+
+### Manually clean up temp files
+
+Due to file locking issues, Bnd on Windows will always copy bundles into a newly launched application rather than referencing them (see [-runnoreferences](/instructions/runnoreferences.html)). This copy can run to a few hundred megabytes.
+
+Again due to file locking (and its overuse of `System.exit()`), Bnd is not very good at cleaning up temporary files on Windows after it exits. So usually, the application folders are left behind after the launched application exits.
+
+A few hundred megabytes per launch starts to add up, and if you're running off a relatively small SSD you can find yourself running out of disk space.
+
+It is recommended that you periodically clear out these files from your temp directory. By default, the temp directory is in `%UserHome%\AppData\Local\Temp`. OSGi temp dirs created by Felix typically have the pattern: `osgi.nnnnnnnnnnnnnnnnn.fw`. Depending on which OSGi framework and which Java utilities you are using in your app, there may be others to look out for - if you are using a custom temp dir (see below), then you can simply clear out the entire directory.
+
+### Turn off/filter real-time malware protection
+
+You can turn off real-time malware protection in Windows settings. However, Windows 10 will only allow this situation to persist for a short time (perhaps 24 hours) before automatically turning it back on again. Also, this approach leaves your entire system without any real-time protection.
+
+A better solution is to disable antivirus scanning for specific directories. You can find instructions on how to do this [here](https://www.windowscentral.com/how-exclude-files-and-folders-windows-defender-antivirus-scans). For maximum benefit, you should exclude:
+* your Bnd workspace;
+* the corresponding Eclipse workspace; and
+* your temp directory (by default: `%UserHome%\AppData\Local\Temp`).
+However, excluding your temp directory might be a bit of a security hole as it is likely to be a place where malware will try and write first. If you are concerned about this, see the next section.
+
+It is possible that excluding other directories may help performance - eg, your `.m2` directory. You need to weigh the merits of excluding each folder vs the incremental security risk.
+
+### Create a custom temp directory
+
+As noted above, for best performance you should exclude the temp directory, however for best safety you may not wish to. A good compromise is:
+
+* Create a temp directory somewhere else on your machine that is specifically for your development work (eg, `C:\temp`).
+* Exclude this directory from antivirus scanning.
+* Configure Eclipse to use use this directory by adding the `-Djava.io.tmpdir=<path to tmp>` vm arg to the `eclipse.ini` file.
+* Configure your launches' `.bndrun` files to use this temp directory by using the `-runvm` instruction (eg: `java.io.tmpdir=C:\\temp` - noting the double-backslash.
```

Comment:
I think the `-runvm` instruction value will also need the `-D` prefix too. `-runvm: -Djava.io.tmpdir=...`

Reply:
From the docs for `-runvm`:

> Additional arguments for the VM invocation. Keys that start with a - are added as options, otherwise they are treated as -D properties for the VM.

To be honest, it's been a while since I actually tried it, but if it doesn't work that way we need to update the docs.

Reply:
I do not see anywhere in the code where that "magic" happens. Also, https://bnd.bndtools.org/instructions/runvm.html, does not contain any such text regarding that "magic" (that whole page is just a place holder). So I don't know what doc for `-runvm` to which you are referring.

Reply:
I'm quoting the instruction index page rather than the specific page for runvm itself.

https://bnd.bndtools.org/chapters/825-instructions-ref.html

Reply:
That doc is still wrong :-)

Reply:
Thanks for confirming - I'll fix both!